pkg/ccl/changefeedccl/changefeedbase/options.go

```diff
@@ -97,6 +113,10 @@ const (
 
 	OptInitialScanOnly = `initial_scan_only`
 
+	OptNoInitialScanType   InitialScanOptType = `no`
+	OptInitialScanType     InitialScanOptType = `yes`
+	OptInitialScanOnlyType InitialScanOptType = `only`
```

Comment:
do we really need these?

Reply:
Yeah; just parse ... strings.ToLower()....

Reply:
I guess... You could keep it to be consistent... But I just don't know how much value this provides.  We parse strings into already descriptive enum value -- and so, having a public option representing string value... probably less useful.

Reply:
yeah I can see how these variables can be a bit redundant, I'll remove them